ExampleMod/Content/Items/ExampleSoul.cs

```diff
@@ -16,8 +16,8 @@ public class ExampleSoul : ModItem
 			// Registers a vertical animation with 4 frames and each one will last 5 ticks (1/12 second)
 			// Reminder, (4, 6) is an example of an item that draws a new frame every 6 ticks
 			Main.RegisterItemAnimation(Item.type, new DrawAnimationVertical(5, 4));
-			
-			ItemID.Sets.AnimatesAsSoul[Item.type] = true; // Makes the item have a 4-frame animation while in world (not held.)
+			ItemID.Sets.AnimatesAsSoul[Item.type] = true; // Makes the item have an animation while in world (not held.). Use in combination with RegisterItemAnimation
+
```

Comment:
This set is legacy as hell. The old comment is true - the animation is hardcoded to 4 frames.